code/modules/mob/living/simple_animal/guardian/guardian_abilities.dm

```diff
@@ -0,0 +1,878 @@
+//SABAKU NO WA ORE STANDO DA
+
+/datum/guardian_abilities
+	var/toggle = FALSE
+	var/mob/living/simple_animal/hostile/guardian/stando = null
```

Comment:
if you're going to meme atleast get it right, "st" is not valid in japanese, what you're looking for is "sutando"
That or just change it to "guardian"

Either or, but do one of them.